### PR TITLE
Fixes #587 :Removed bugged links

### LINKED ISSUE #587: Bugged Pinterest links
### Expected results
All working links
### Actual results
Pinterest link are not to any profile
### Steps to reproduce the issue
remove them
## Would you like to work on this
yes

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #587
# Things Done

### CODE REVIEW

**Comment by saarthakchats** [comment]
https://saarthakchats.github.io/crappy/

File: _includes/footer.html [hunk 1]
```diff
@@ -53,11 +53,7 @@
 										<i class="fa fa-vk" aria-hidden="true"></i>
 									</a>
 								</li>
-								<li>
-									<a href="https://in.pinterest.com/pin/390476230184177529/">
-										<i class=" ti-pinterest" aria-hidden="true"></i>
-									</a>
-								</li>
+
```
Comment (NaviRocker) [inline _includes/footer.html]: Why is this extra line?

**Comment by saarthakchats** [comment]
The link was bugged, no profile existed

**Comment by saarthakchats** [comment]
@naveentherocker

File: _includes/footer.html (hunk 1 above)
Comment (vaibhavsingh97) [reply _includes/footer.html]: @saarthakchats Please remove this blank line

File: _includes/footer.html [hunk 2]
```diff
@@ -171,11 +167,6 @@ <h6 class="title">Instagram</h6>
 										<i class="fa fa-vk" aria-hidden="true"></i>
 									</a>
 								</li>
```
Comment (vaibhavsingh97) [inline _includes/footer.html:168]: same

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

File: _includes/footer.html (hunk 2 above)
Comment (vaibhavsingh97) [reply _includes/footer.html:168]: There is blank line left here

**Comment by saarthakchats** [comment]
@vaibhavsingh97 Pls merge

**Review by vaibhavsingh97 — APPROVED** [review]

**Pull request merged**